src/iOS.Core/Controllers/LoginAddViewController.cs

```diff
@@ -113,6 +135,42 @@ public override void ViewDidLoad()
             folderNames.Insert(0, AppResources.FolderNone);
             FolderCell.Items = folderNames;
 
+            _personalOwnershipPolicyApplies = _policyService.PolicyAppliesToUser(PolicyType.PersonalOwnership).GetAwaiter().GetResult();
+            var index = 0;
+            if (_personalOwnershipPolicyApplies)
+            {
+
+                _organizations = _organizationService.GetAllAsync().GetAwaiter().GetResult().OrderBy(o => o.Name).ToList();
+
+                OrganizationCell.Items = _organizations.Select(o => o.Name).ToList();
+                OrganizationCell.ValueChanged += Type_ValueChanged;
+
+                _collections = _collectionService.GetAllDecryptedAsync().GetAwaiter().GetResult();
```

Comment:
🤔 Could you test if `_collectionService.GetAllDecryptedAsync` takes too long the app crashes? You can mock this by just adding a task delay of some seconds in the first line inside `GetAllDecryptedAsync()` which will be mocking the case of a really large number of collections to decrypt.
We shouldn't be using `GetAwaiter().GerResult()` and just put some loading on the view and performing the work asynchronously updating the table view afterwards but well I know we already have other calls that use that like `Folders`; so I guess it's "fine" for now as long as this loads "fast" and doesn't crash the app because `ViewDidLoad` took to long to finish.